### DISCUSSION TITLE
What extensions and themes are you using with OpenMage?

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1900] [Extensions & Modules]
This post aims to provide information about paid and free extensions and themes that still work with OpenMage.

**Please provide the following**

- OpenMage version used in production
- Server configuration (OS, Apache / Nginx, PHP, MySQL, Redis, Varnish) and what versions
- Short description of the extension/theme
- If the developer of the extension/theme still sells it and if it provides technical support or not

Thank you all.

**Comment by aterjung** [comment]
AOE_Scheduler  
PHP7.4 :white_check_mark: 
PHP 8 :question:

- Version latest master
- PHP 7.2 - 7.4
- advanced scheduler module

latest master branch is needed to work with PHP7.4, did not test ist with PHP8.

**Comment by luigifab** [comment]
For our websites and for websites we worked: latest release of OpenMage 20, without Mysql4 classes, without admin router compatibility, without scriptaculous, and with some kind of pending PR. Since some days with the public directory. Some of them without wysiwyg editor and Mage_Persistent+Mage_Paypal (full removal). Server:
- Debian
- Nginx
- PHP 8.0 (previously 7.4)
- MariaDB 10.7
- Redis 5.0 (session and files)
- Varnish 6.4

We are currently using (not all here):
- [Allopass/Hipay](https://github.com/hipay/hipay-fullservice-sdk-magento1): payment methods
- [Alma/Installments](https://github.com/alma/alma-installments-magento1): payment in 2/3/4 times by credit card
- [AMG/Sentry](https://github.com/getsentry/magento-amg-sentry-extension): connector to Sentry to track PHP errors (I will extract and push to a new repo our changes)
- [Aoe/DbRetry](https://github.com/AOEpeople/Aoe_DbRetry)
- [CVM/GoogleTagManager](https://github.com/CVM/Magento_GoogleTagManager)
- [Fooman/Surcharge](https://fooman.com/magento-extension-surcharge.html): payment fees
- [IntegerNet/RemoveCustomerAccountLinks](https://github.com/integer-net/RemoveCustomerAccountLinks): remove customer account links
- [Kyrena/Shippingmax](https://github.com/kyrena/openmage-shippingmax): pick up shipping methods (Chrono Relais, Colis Privé, Mondial Relay, DPD FR, InPost IT GB PL, Pocztex, Packeta/Zásilkovna, Boxberry, 5post, PickPoint, Shiptor, Store delivery, Store locator)
- [Luigifab/Apijs](https://github.com/luigifab/openmage-apijs): multithread generation of products/categories/wysiwyg images with python-pil
- [Luigifab/Cronlog](https://github.com/luigifab/openmage-cronlog): show history and configuration of cron jobs
- [Luigifab/Maillog](https://github.com/luigifab/openmage-maillog): show history of emails sent and sync customers with Mautic and picture tag for images and if conditions for emails templates
- [Luigifab/Urlnosql](https://github.com/luigifab/openmage-urlnosql): products url without core_url_rewrite
- [Lyranetwork/Payzen](https://payzen.io/fr-FR/plugins/index.html): payment methods
- [Nexcessnet/Turpentine](https://github.com/nexcess/magento-turpentine): varnish cache (I will extract and push to a new repo our changes)
- [Owebia/Shipping2](https://github.com/owebia/magento1-module-advanced-shipping): shipping methods
- Tangkoko/SecuredPdf: copyright notice for downloadable pdf

We have updated some of them for PHP 8.0.

**Reply by sreichel** [reply]
> * [IntegerNet/RemoveCustomerAccountLinks](https://github.com/integer-net/RemoveCustomerAccountLinks): remove customer account links

This feature was added with #321. (via layout update, no system config)

**Reply by luigifab** [reply]
We using it for system config.

**Comment by addison74** [comment]
I am managing Magento versions 1.9.2.4 and 1.9.4.5 and OpenMage 20.0.13. All shopping carts are installed on dedicated servers with Debian 9 and 10, Apache, PHP 7.0 and 7.4, Percona SQL Server. All logs are activated but I did not find any issues over the time. In some configurations I am using 1 - 2 databases and load balancers (HAProxy, Pound) with TSL 1.3 termination (Envoy, Pound).

Here are the extensions I am currently using without any issues:

**Activo CatalogSearch** - this improves the search results, a must have in all installations. It is not maintained and the latest version I found searching on the Internet is 1.0.7. => ```Update: This is no longer needed for OpenMage. We can switch from Backend for AND/OR in search queries```

**Hackathon HoneySpam** - to deal with spam in Magento forms (newsletter, contact). It is doing a great job. I reported a few issues on GitHub and they were solved by the contributors.

**MobWeb AnalyticsSalesFunnel**

**MobWeb MultipleBankTransferPaymentMethods** - for adding more payments methods.

**Nexcess Turpentine** - for using Varnish Cache 5 + Pound (only one store)

**AheadWorks PriceMatch** - for matching prices. The extension is not maintained or offered. I customized it a lot.

**AheadWorks Zblocks** - for creating customizable blocks. The extension is not maintained or offered. I customized it a little.

**MageWorx Product Custom Options** - still offered and maintained. This is a great extension when you don't want to create configurable products (for speed improvement). The extension still has bugs for some scenarios but I did not report them yet. The team is doing a fair technical support when needed.

**EM Productfilterwidget** - it is no longer offered and maintained. This is a good extension for creating blocks based on filters like special time, manufactures, categories, price, ...). I had to customize it a lot because what I got from the developer did not work and he did not respond to my complains.

 **Prasan CustomerNavigationLinks** - this is for adding links in customers accounts like Logout link.

**Fballiano OpenMage Image Cleaner** - the best cleaner for orphan images (all kinds of images). It was a pleasure to work with Fabrizio on this project and it replaced all my Defcon ImaClean installations.

**Averun Sizechart** - the best size chart for Magento 1 & 2. The developer still offers this extension and fixes the reported bugs. Always a pleasure to work with them.

**AveRun Color Swatch for Products with Custom Options** - the best for adding one image to your custom options. It needs customization but it is easy to manage and remove. The developer still offers this extension and fixes the reported bugs. Always a pleasure to work with them.

I also used AddThis SharingTool, EcommerceTeam CloudZoom, Inchoo PHP7 (for Debian 9 with PHP 7.0 and Magento 1.9.2.4), Thirty4 CatalogSale (creates blocks with special prices), Kanavan Searchautocomplete, Sashas Randnew (for fixing an issue in new products block).

The list is longer than that over the years I tested and reported a lot of extensions. As themes I installed variants of Fortis, Porto and themes from Template Monsters.
